modcc/printer/cexpr_emit.hpp

```diff
@@ -60,9 +60,9 @@ class SimdExprEmitter: public CExprEmitter {
 
 protected:
     static std::unordered_set<std::string> mask_names_;
-    bool processing_true_;
-    bool is_indirect_;
-    std::string current_mask_, current_mask_bar_, input_mask_;
+    bool processing_true_ = false;
+    bool is_indirect_ = false;
+    std::string current_mask_ = "", current_mask_bar_ = "", input_mask_ = "";
```

Comment:
Strings do not need to be explicitly initialized. Was this a warning?

Reply:
Actually, just `processing_true_` _needs_ to be initialised to avoid an uninitialised read. However, I added the rest for consistency.

Reply:
`bool` members really should be initialized, I agree. I think we can leave the strings alone though, because we can be sure they won't ever be a problem.

Reply:
As a matter of preference I do initialise all members, but yes, unneeded and I can remove it.